xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -819,6 +865,26 @@ void CGUIDialogVideoInfo::OnGetFanart()
   Update();
 }
 
+void CGUIDialogVideoInfo::OnSetUserrating()
```

Comment:
Correct, will change. Was quiet torn between it beeing "user rating" or "userrating"